app/helpers.php

```diff
@@ -117,12 +117,17 @@ function human_diff_string(\Carbon\Carbon $ts1, \Carbon\Carbon $ts2, $absolute =
         return 'unknown length';
     }
 
+    if ($ts2 <= \Carbon\Carbon::minValue() || $ts2 >= \Carbon\Carbon::maxValue()
+        || $ts1 <= \Carbon\Carbon::minValue() || $ts1 >= \Carbon\Carbon::maxValue()) {
+        return 'unreliable dates';
```

Comment:
Is it wise to throw an error here? Doing so would prevent any page from loading since there is no error catching above this function in many cases, I could add a more detailed output to allow the problem to be resolved if needed.